cypress/integration/AboutMeAndLocation.feature

```diff
@@ -0,0 +1,49 @@
+Feature: About me and and location
+  As a user
+  I would like to add some about me text and a location
+  So others can get some info about me and my location
+
+  The location and about me are show on the user. Later it will be possible
+  to search for users by location.
+
+  Background:
+    Given I am logged in
+
+  Scenario: I change my name
+    When I visit the "settings" page
+    And I enter "Hansi" as my name
+    And I press "Save"
+    And I can click on my profile picture in the top right corner
+    Then I can see a "Hansi" as my name
+
+    When I refresh the page
+    And I can click on my profile picture in the top right corner
+    Then I can see a "Hansi" as my name
+
+    When I visit the "settings" page
+    And I enter "Peter Lustig" as my name
+    And I press "Save"
```

Comment:
Remove that scenario

Reply:
Why? @roschaefer

Reply:
The three lines. It does not have an assertion, ie. a `Then` step